packages/app/cypress/e2e/runs.cy.ts

```diff
@@ -107,6 +107,12 @@ describe('App: Runs', { viewportWidth: 1200 }, () => {
 
       cy.findByTestId('sidebar-link-runs-page').click()
 
+      // TODO: investigate the scenario for this test
```

Comment:
@estrada9166 can you please review this test? The new GQL implementation for remote queries (cloud) adds caching, which seems to interfere with this test. Busting the cache at his point as I added passes the test, but it may turn it into an unrealistic test. I'm not sure about the details of this scenario/feature. Can you please take a look?